app/views/posts/index.html.erb

```diff
@@ -1,21 +1,28 @@
-<p id="notice"><%= notice %></p>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Bootstrap Example</title>
```

Comment:
I know this element is not visible on the actual website but where is that "title" supposed to appear?

Reply:
I do not not actually know but it is important to keep this code just in case.